docs/src/tutorials/write_documentation.rst

```diff
@@ -79,9 +79,16 @@ __ https://github.com/PyMoDAQ/PyMoDAQ/
 
 .. note::
 	The important branches of the PyMoDAQ repository are as follow:
-		* **main** is the last stable version. This branch is maintained by the owner of the repository, and we should not send a pull request directly to it.
-		* **pymodaq-dev** is the development branch, which is ahead of the *main* branch (it contains more commits than the *main* branch. External contributions should be send on this branch. The owner of the repository will test all the changes that has been suggested in the *pymodaq-dev* branch before sending them into the *main* branch.
-		* **pymodaq_v3** concerns the version 3.
+		* **4.4.x** is the last stable version (in october 2024 :) if you update the documentation in this branch, it will update the 4.4.x version of the documentation
```

Comment:
maybe add that the stable version is only accepting bug correction while the dev version is where new features will be made

Reply:
and that probably in the future there would be:

* 4.4.x
* 5.0.x
* 5.1.x_dev

where the 4.4 will stay for a long term support

Reply:
i don't know how to add the new commit diff to this conversation. But it tried to include what you wanted. 
May be there is somewhere else i should add the branching rules and naming system ?

Reply:
From the review I'm seeing all new changes since my last review. For me this is fine like this. Are you done with the modification you wanted to do? Is yes, I'll accept the PR

Reply:
@loicguilmard ?